### PR TITLE
docs for single-run range backfills

### PR DESCRIPTION
**Pull request by sryza**

### CODE REVIEW

**Review by clairelin135 — COMMENTED** [review]
I've also seen multiple users ask how to kick off a run via a schedule that covers a range of partitions.

If you get the chance, I think it would be nice to add docs for that too.

File: docs/content/concepts/partitions-schedules-sensors/backfills.mdx [hunk 1]
```diff
@@ -33,6 +33,61 @@ width={1737}
...
+
+Dagster supports backfills that execute as a single run that covers a range of partitions. For example, this allows you to execute the backfill as a single Snowflake query. After it completes, Dagster will track that all those partitions have been filled.
+
+For this to work, your code needs to be written in a way that allows it to operate on partition ranges, instead of just single partitions. If you're using any of the built-in database I/O manager, like [Snowflake](/integrations/snowflake), [BigQuery](/integrations/bigquery), or [DuckDB](/\_apidocs/libraries/dagster-duckdb), you'll have this out-of-the-box.
+
+If your code uses any of the `partition_key`, `asset_partition_key`, `asset_partition_key_for_output`, or `asset_partition_key_for_input` context methods or properties, you'll need to update it to use methods or properties like `asset_partition_time_window`, `asset_partition_key_range`, `asset_partition_keys`, `asset_partition_time_window_for_output`, `asset_partition_key_range_output`, `asset_partition_time_window_for_input`, or `asset_partition_key_range_for_input` instead.
```
[the remark this answers is not in the document]
Comment (sryza) [reply docs/content/concepts/partitions-schedules-sensors/backfills.mdx:44]: Thanks for catching

**Comment by sryza** [comment]
> I've also seen multiple users ask how to kick off a run via a schedule that covers a range of partitions.

> If you get the chance, I think it would be nice to add docs for that too.

I'm going to save this for a later PR

File: docs/content/concepts/partitions-schedules-sensors/backfills.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (clairelin135) [reply docs/content/concepts/partitions-schedules-sensors/backfills.mdx:44]: Looks like this part still needs to be updated? Maybe need to push these changes

[the remark this answers is not in the document]
Comment (sryza) [reply docs/content/concepts/partitions-schedules-sensors/backfills.mdx]: Oops, you are right. Just pushed.

**Pull request merged**